CMSgov/design-system
# [WNMGDS-2020] Add a11y guidance for HCgov alert

## Summary

- Added minor guidance the alert component that had not been moved into the doc site

## How to test

- run the doc site with `yarn start` and see that the alert page has addition a11y content for the HCgov theme. 

## Checklist




### If this is a change to code:


### If this is a change to documentation:
